src/Movim/Template/Builder.php

```diff
@@ -138,55 +139,31 @@ public function title(): string
      */
     public function language(): string
     {
-        $lang = $this->user && $this->user->language
-            ? $this->user->language
-            : Configuration::get()->locale;
-
-        if (empty($lang)) {
-            return 'en';
+        if ($this?->user?->language != null) {
+            $this->lang = $this->user->language;
+            return \Movim\i18n\Locale::printISO639($this->lang);
         }
 
-        // String casing for the web
-        $split = array_slice(preg_split('/[_-]/', $lang), 0, 3);
-        $lc = array_shift($split);
-        $langCode = strtolower($lc);
+        $l = \Movim\i18n\Locale::start();
```

Comment:
I'm trying to use more descriptive variable names, you can name it $locale there :)